lib/customers.js

```diff
@@ -88,6 +88,10 @@ function update (id, data, userToken, txId) {
     ' where id=$1 returning *'
 
   return db.one(sql, [id])
+     .then(customerData => {
+       return getEditedData(id)
+         .then(customerEditedData => selectLatestData(customerData, customerEditedData))
+     })
```

Comment:
```suggestion
     .then(customerData => getEditedData(id))
     .then(customerEditedData => selectLatestData(customerData, customerEditedData))
```

Reply:
Hmm, not sure if this works after removing the cascading promise though. You'll need to check if you make this change